### PR TITLE
DDF-1120

### PR DESCRIPTION
**Pull request by lessarderic**
https://tools.codice.org/jira/browse/DDF-1120

### CODE REVIEW

**Comment by shaundmorris** [comment]
The Travis build should be fine (oracle jdk 7 built fine). This is known issue with JDKs in the Travis build. I thought we had it documented to include the JIRA ticket number in the commit messages, but apparently not. I added an example here: https://tools.codice.org/wiki/display/DDF/Community+Contributions

File: test/pom.xml [hunk 1]
```diff
@@ -42,6 +41,17 @@
 		</repository>
 	</repositories>
 
+	<dependencies>
```
Comment (pklinef) [inline test/pom.xml]: Instead of adding dependencies, can you try adding the test scope to dependency management?

**Comment by pklinef** [comment]
Looks like your commit messages are missing a DDF ticket reference.

File: test/pom.xml (hunk 1 above)
Comment (lessarderic) [inline test/pom.xml]: Tried that but it didn't work so I had to keep the dependency elements. Let me know if you want me to try something else.

**Comment by lessarderic** [comment]
Yeah, Shaun noticed that too. Is there a way to fix that now that I've push my changes and submitted the pull request?

**Comment by kcwire** [comment]
@lessarderic since this is your branch you should be able to do `git commit --amend -m "New commit message"` and force push.

**Comment by shaundmorris** [comment]
Thanks for the tip @kcwire I should have posted that in my original comment. It is looking good @lessarderic

**Comment by lessarderic** [comment]
Thanks guys. With some help from Scott, we were able to combine the two commits and update the message.

File: test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java [hunk 2]
```diff
@@ -14,19 +14,6 @@
  **/
 package ddf.catalog.test;
 
-import org.junit.After;
```
Comment (pklinef) [inline test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java]: It doesn't look like you need this import reordering for this fix.  Can you remove TestCatalog from your commit?

Comment (lessarderic) [inline test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java]: After fixing the compile issue I noticed that some of the imports weren't needed (getting warnings in Eclipse) so I cleaned them up. And that's actually why I had two separate commits initially, one for the pom change and one for the import clean up. Do you want me to undo the import cleanup altogether or create a separate commit for it?

Comment (pklinef) [inline test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java]: I did not notice the removed imports.  It is fine how it is.

**Pull request merged**